lib/src/widgets/clock.dart

```diff
@@ -287,7 +287,8 @@ class _CountdownClockState extends State<CountdownClockBuilder> {
   void didUpdateWidget(CountdownClockBuilder oldClock) {
     super.didUpdateWidget(oldClock);
 
-    if (widget.clockUpdatedAt != oldClock.clockUpdatedAt) {
+    if (widget.timeLeft != oldClock.timeLeft ||
```

Comment:
I don't think that change is a good idea. The way to clock is designed is to rely only on `clockUpdatedAt` to know when the clock has changed. We want to force update `clockUpdateAt`, and here you'd broke that contract.

Reply:
Not sure to understand your comment. Do you mean that the only to change the clock should be the `clockUpdateAt` field ?

Reply:
The only way to indicate the clock has changed is to update `clockUpdatedAt`. Meaning if you want to update `timeLeft` you *have* to update `clockUpdatedAt`. We want to support updating the clock with the same time left, this is the only way.
If you don't provide at all the `clockUpdatedAt` it means `timeLeft` will never change.

Reply:
You can read the tests in `clock_test.dart` to understand also, it contains the specification.

Reply:
I don't understand why you are saying that `We want to support updating the clock with the same time left, so this is the only way to ensure it.`

With what I have done you can still update the clock with the same time left.

Reply:
The reason I needed to also update the clock when `timeLeft` change (which makes sense I think) is that on the broadcast game screen when going through the moves I need to display the clocks and the clocks are always inactive with no `clockUpdatedAt` expect for the current broadcast path if the game is still ongoing.

Reply:
What I could do instead is simply use a text widget but then why add an `inactive` field.